index.html

```diff
@@ -106,8 +106,8 @@ <h4>Free Educations</h4>
                             <div class="row">
                                 <div class="col-md-12">
                                     <div class="slide_text">
-                                        <h3><span span class="theme_color">You only have know one thing</span><br>you can learn anything</h3>
-                                        <h4>Free Educations</h4>
+                                        <h3><span span class="theme_color">You only have to know one thing</span><br>you can learn anything</h3>

```

Comment:
This is a good start 😁. Please make sure you fix the formatting.

Reply:
Thank you! fix the formatting as in what should i do?

Reply:
span should be on the next line since its enveloped by h3

Reply:
Oh okay ill do that